server/app/services/ti/DateOfBirthExistsException.java

```diff
@@ -0,0 +1,7 @@
+package services.ti;
+
+public class DateOfBirthExistsException extends java.util.NoSuchElementException {
+  public DateOfBirthExistsException() {
```

Comment:
Does this constructor brings any value?

Reply:
Removed this Exception as the PM wants this to be an editable field..